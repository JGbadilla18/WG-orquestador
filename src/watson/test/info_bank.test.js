const testWatson = require('./testWatson');

watsonText = 'Contacto del banco'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
