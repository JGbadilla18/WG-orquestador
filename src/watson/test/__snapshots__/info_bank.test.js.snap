// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contacto del banco 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Puedes contactarnos por estos medios: 
  *  PBX: 1717
  *  Número directo: 2420-3000
  *  Número directo: 2411-6000
  *  Whastapp: 2411-6000
  *  Bi Chat: https://www.corporacionbi.com/gt/bancoindustrial/bichat
  *  Correo: serviciobi@bi.com.gt",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
